src/refactor/index.jsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import './index.css';
+
+
+// return on each Click the value of Square
+function Square(props) {
+  return (
+    <button className="square" onClick={props.onClick}>
+      {props.value}
+    </button>
+  )
+}
+// Describe Board Game + set boolean to setup user 
+class Board extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      squares: Array(9).fill(null),
+      xIsNext: true,
+    };
+  }
+
+  // handle which turn is it
+  handleClick(i) {
+    const squares = this.state.squares.slice();
+    squares[i] = this.state.xIsNext ? 'X': 'O';
+    this.setState({
+      squares: squares,
+      xIsNext : !this.state.xIsNext,
+    });
+    
+}
+
+
+  renderSquare(i) {
+    return (
+      <Square
+        value={this.state.squares[i]}
+        onClick={() => this.handleClick(i)}
+      />
+    );
+    
+   
+  }
+  
+  render() {
+    
+    const winner = calculateWinner(this.state.squares);
+    let status;
+    if (winner) {
+      status = 'Winner:' + winner
+    } else {
+      status = 'Next player:' + (this.state.xIsNext ? 'X' : 'O');
+    }
```

Comment:
Good decisions, first, deciding to use one status fit the business logic of the app, while there is a winner there is no need for the next player status, second, you are using conditions in a good and readable way, the triangle fit a small case where you have to options to assign, a smart ass dev would do a triangle condition within a triangle condition like:
```
const status = !!winner ? `Winner: ${winner}` : `Next player: ${this.state.xIsNext ? 'X' : 'O' }`
``` 
But in my opinion, this style is bad for readability and just seems sophisticated. 
p.s- Always prefer string templates over the concatenation of the string with a variable.